dir/fs.go

```diff
@@ -1,156 +1,50 @@
 package dir
 
 import (
-	"errors"
 	"io/fs"
 	"os"
-	"path"
 	"path/filepath"
 )
 
-// RootedFS is a file system interface supporting for getting root path.
-type RootedFS interface {
+// SysFS is virtual file system interface that support fs.FS and SysPath method.
+type SysFS interface {
 	fs.FS
-	// Root returns the root path of the RootedFS
-	Root() string
+
+	// SysPath returns the real system path of the given path items in the SysFS.
+	SysPath(items ...string) (string, error)
 }
 
-type rootedFS struct {
+type sysFS struct {
 	fs.FS
 	root string
 }
 
-// Root returns the root path of the rootedFS
-func (r *rootedFS) Root() string {
-	return r.root
+// SysPath returns the real system path of the given name in the SysFS.
+func (s sysFS) SysPath(items ...string) (string, error) {
+	pathItems := []string{s.root}
+	pathItems = append(pathItems, items...)
+	return filepath.Join(pathItems...), nil
 }
 
-// NewRootedFS create a rootedFS
+// NewSysFS returns the SysFS for the given roots directories.
 //
-// if fsys is nil, uses os.DirFS
-func NewRootedFS(root string, fsys fs.FS) RootedFS {
-	if fsys == nil {
-		return &rootedFS{FS: os.DirFS(root), root: root}
+// Support one root directory for rc.1, and may support union directories FS
+// after rc.1.
+func NewSysFS(roots ...string) SysFS {
```

Comment:
How about 
```go
func NewSysFS(root string) SysFS
```

For "union directories FS after rc.1", maybe we can have something like
```golang
func NewUnionFS(roots ...string) (SysFS, error)
```

Reply:
Updated  `func NewSysFS(root string) SysFS`